postgres/osfamilymap.yaml

```diff
@@ -63,11 +63,12 @@ RedHat:
     gpgkey: 'https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-{{ release }}'
     baseurl: 'https://download.postgresql.org/pub/repos/yum/{{ repo.version }}/redhat/rhel-$releasever-$basearch'
 
-{%- if grains.get('osmajorrelease', 0) >= 7 %}
   pkgs_deps:
     - libicu
+    {%- if grains.get('osmajorrelease', 0) == 7 %}
+    - systemd-sysv
+    {%- endif %}
```

Comment:
Only need this package on all RedHat 7 platforms (the equivalent change for Amazon Linux 2 is [done above](https://github.com/saltstack-formulas/postgres-formula/pull/315/files#r665291631)).